code/game/gamemodes/wizard/wizard.dm

```diff
@@ -248,6 +248,20 @@
 		qdel(spell_to_remove)
 		mind.spell_list -= spell_to_remove
 
+/datum/mind/proc/add_spell(var/obj/effect/proc_holder/spell/spell) //To add a specific spell to a mind

```

Comment:
As said, we don't need another proc to add spells, just use this one.
https://github.com/tgstation/-tg-station/blob/master/code/datums/mind.dm#L1451